src/connectors/hierarchical-menu/connectHierarchicalMenu.js

```diff
@@ -213,6 +213,43 @@ export default function connectHierarchicalMenu(renderFn, unmountFn) {
 
         return nextState;
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`